flow/build.gradle

```diff
@@ -40,5 +40,8 @@ project.tasks.register("engineJar", Jar) {
 }
 
 project.tasks.named('engineJar').configure { dependsOn(project.engineClasses) }
-project.tasks.named('assemble').configure { dependsOn(project.tasks.engineJar) }
-project.tasks.named('module').configure { dependsOn(project.tasks.engineJar) }
+
+// The server build does not need flow_engine.jar
```

Comment:
I think it does actually need the `engineJar`, as currently assembled. When I tried to simply remove this before, I found that the module code references some classes that are in the engine. That could could possibly be moved to the API jar for this module, but I don't know enough about the workings here to say anything with confidence.

Reply:
Hmm, I ran the flow tests after deleting the jar and I think it worked fine...  I'll check again.

Reply:
TeamCity is definitely not happy, though.

Reply:
thank you for the help!